spec/datadog/core/configuration/components_spec.rb

```diff
@@ -1380,6 +1384,24 @@
         end
       end
 
+      context 'when the appsec is re-used' do
+        include_context 'replacement' do
+          let(:appsec) { components.appsec }
+        end
+
+        it 'shuts down all components but the tracer' do
```

Comment:
If you do decide to keep the replacement support (don't :P), this test description is incorrect ;)

But better yet -- delete all this! :D